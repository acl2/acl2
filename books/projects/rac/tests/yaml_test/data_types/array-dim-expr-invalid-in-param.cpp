// RAC begin

template <int n>
int foo(int a[n]) {
  return a[4];
}
