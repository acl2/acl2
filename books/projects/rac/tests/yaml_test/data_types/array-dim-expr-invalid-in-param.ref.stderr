array-dim-expr-invalid-in-param.cpp:4 (14-15): Invalid array size (it shoud be a constant, stricly positive expression)
 4 | int foo(int a[n]) {
   |               ^
